ocs_ci/utility/kms.py

```diff
@@ -35,6 +35,7 @@
     get_default_if_keyval_empty,
     get_cluster_name,
 )
+from subprocess import CalledProcessError
```

Comment:
This is already imported in line 13 above

Reply:
removed